pkgs/development/compilers/gcc/common/configure-flags.nix

```diff
@@ -27,7 +27,7 @@
 , disableBootstrap ? stdenv.targetPlatform != stdenv.hostPlatform
 }:
 
-assert disableGdbPlugin -> !enablePlugin;
+assert !enablePlugins -> disableGdbPlugin;
```

Comment:
I suggest leaving out renaming part and just update the assert for the same reasons as in https://github.com/NixOS/nixpkgs/pull/217978#pullrequestreview-1312721329